sync/src/types/mod.rs

```diff
@@ -249,11 +249,24 @@ impl HeadersSyncController {
     }
 }
 
+#[derive(Clone, Copy, Eq, PartialEq, Debug)]
+pub enum Version {
+    Old,
+    New,
+}
```

Comment:
> A public type called Version is easy to misunderstand, protocol version, implementation version, binary version, or else?

https://github.com/nervosnetwork/ckb/pull/2765#discussion_r660292829

Reply:
change to binary version